### ISSUE TITLE
Repeat tag X number of times

### ISSUE THREAD

**Issue by dacap** [aseprite/aseprite#1989]
Add a field to tags to loop it X number of times (and when X=0 it's the current behavior=repeat infinite times).

This feature was requested several times, I'm making this one to group everything:

* https://github.com/aseprite/aseprite/issues/1275
* https://github.com/aseprite/aseprite/issues/1740
* https://community.aseprite.org/t/loop-section-x-times-then-continue/1910
* https://community.aseprite.org/t/repeat-cels-looping-backgrounds-concept/844
* https://community.aseprite.org/t/add-a-loop-option-in-frame-tags/1565/2
* https://twitter.com/johanwendin/status/1025384484793982978
* https://steamcommunity.com/app/431730/discussions/1/3300476731154878695/

And maybe in other places.

**Comment by slmjkdbtl** [comment]
Thanks for taking this into account! Maybe just a bool button and a key that's written into the json file for now if it's difficult to visualize in the editor. For my particular case I just need the json file to have the `loop` data so I don't have to manually set it in the game code for every sprite.

**Comment by Geokureli** [comment]
TBH I find this field very confusing

In the video below, enabling "repeat" via the checkbox makes the animation play once (as in it does not repeat), disabling repeat makes it repeat, infinitely. Furthermore you can't put 0, it will be changed to infinity

https://github.com/aseprite/aseprite/assets/2609513/ce9b875e-30ff-407d-a6ff-f2f844ff896e

it says "repeat" but it actually seems to indicate the number of plays.

The main reason I bring this up is that in the next release of HaxeFlixel, our Aseprite atlas importer will look at the tags' `repeat` field to determine whether animations should loop, and in it's current form it's not very intuitive for this purpose. I realize this isn't exactly *your* problem, and maybe there's a better solution for flixel (maybe a plugin?), but i still feel like this could be less confusing, and less restrictive

**Comment by dacap** [comment]
> it says "repeat" but it actually seems to indicate the number of plays.

That's right. The field was designed to specify "how many times this tag should be repeated/played". You can use "1" to repeat the tag one time (no loops), or any N value to repeat it N times. The "0" is a special value which means "infinite times" (because playing/repeating an animation 0 times doesn't make sense).

Flixel could choose to use this field as they prefer (0 = infinite plays as the Aseprite UI, 1 = no loops = just play once, and other value play N times).

Probably the label could be replaced with other thing "Play N times" if the confusion is persistent between users.

**Comment by Geokureli** [comment]
I don't have any great solutions, here. Just wanted to express my confusion in someone else had any great ideas
